modules/ztp-image-based-upgrade-upgrade.adoc

```diff
@@ -128,4 +128,60 @@ cgu-ibu-finalize        30h   Completed   All clusters are compliant with all th
 cgu-ibu-prep            31h   Completed   All clusters are compliant with all the managed policies
 cgu-ibu-upgrade         31h   Completed   All clusters are compliant with all the managed policies
 ----
---
\ No newline at end of file
+--
+
+. You can remove the OADP Operator and its configuration files after a successful upgrade.
+
+.. Change the `complianceType` to `mustnothave` for the OADP Operator namespace, Operator group, and subscription in the `common-ranGen.yaml` file.
++
+[source,yaml]
+----
+- fileName: OadpSubscriptionNS.yaml
+  policyName: "subscriptions-policy"
+  complianceType: mustnothave
+- fileName: OadpSubscriptionOperGroup.yaml
+  policyName: "subscriptions-policy"
+  complianceType: mustnothave
+- fileName: OadpSubscription.yaml
+  policyName: "subscriptions-policy"
+  complianceType: mustnothave
+- fileName: OadpOperatorStatus.yaml
+  policyName: "subscriptions-policy"
+  complianceType: mustnothave
+----
+
+.. Change the `complianceType` to `mustnothave` for the OADP Operator namespace, Operator group, and subscription in the site `PolicyGenTemplate` file.
++
+[source,yaml]
+----
+- fileName: OadpSecret.yaml
+  policyName: "config-policy"
+  complianceType: mustnothave
+- fileName: OadpBackupStorageLocationStatus.yaml
+  policyName: "config-policy"
+  complianceType: mustnothave
+- fileName: DataProtectionApplication.yaml
+  policyName: "config-policy"
+  complianceType: mustnothave
+----
+
+.. Merge the changes with your custom site repository and wait for the ArgoCD application to synchronize the change to the hub cluster. The status of the `common-subscriptions-policy` and the `example-cnf-config-policy` policies change to `Non-Compliant`.
+
+.. Apply the change to your target clusters by using the {cgu-operator-full}. For more information about rolling out configuration changes, see the "Update policies on managed clusters" section.
+
+.. Monitor the process. When the status of the `common-subscriptions-policy` and the `example-cnf-config-policy` policies for a target cluster are `Compliant`, the OADP Operator has been removed from the cluster. Get the status of the policies by running the following commands:
++
+[source,terminal]
+----
+$ oc get policy -n ztp-common common-subscriptions-policy
+----
+
```

Comment:
Remove "+" to fix indentation issue:

![Screenshot from 2024-07-31 11-45-19](https://github.com/user-attachments/assets/8e7118b5-ec2f-4145-85cf-52c560d2316d)